README.md

```diff
@@ -47,3 +47,11 @@
 | ---- | ------------- | -------------- | ---------------- | ---------------- | --------------------------------- | ---------------------------------------------------------------------------------------- | -------------- |
 | 1    | Random Forest | 1024-bit ECFP4 | 3.7283 +- 0.0219 | 2.7107 +- 0.0173 | [Mufei Li](mufeili1996@gmail.com) | [Paper](https://www.stat.berkeley.edu/~breiman/randomforest2001.pdf), [Code](./examples) | Feb 4th, 2021  | 
 | 2    | GCN           | GraphConv      | 5.4312 +- 2.4279 | 2.6516 +- 0.1190 | [Mufei Li](mufeili1996@gmail.com) | [Paper](https://arxiv.org/abs/1609.02907), [Code](./examples)                            | Feb 4th, 2021  |
+
+
+### SIDER
+
+| Rank | Model         | Featurization  | Test ROC-AUC     | Validation ROC-AUC | Contact                           | References	                                                                           | Date           |
+| ---- | ------------- | -------------- | ---------------- | ------------------ | --------------------------------- | ---------------------------------------------------------------------------------------- | -------------- |
+| 1    | GCN           | GraphConv      | 0.6265 +- 0.0076 | 0.6409 +- 0.0094   | [Yuanqi Du](ydu6@gmu.edu) | [Paper](https://arxiv.org/abs/1609.02907), [Code](./examples)                            | Feb 10th, 2021 |
+| 2    | Random Forest | 1024-bit ECFP4 | 0.6504 +- 0.0011 | 0.6197 +- 0.0023   | [Yuanqi Du](ydu6@gmu.edu) | [Paper](https://www.stat.berkeley.edu/~breiman/randomforest2001.pdf), [Code](./examples) | Feb 10th, 2021 |
```

Comment:
For ROC-AUC, it should be the larger, the better. The numbers suggest that RF performs better in terms of the test ROC-AUC.